app/javascript/components/enrollment/steps/Address.js

```diff
@@ -14,12 +14,9 @@ class Address extends React.Component {
       // When viewing existing patients, the `monitored_address_state` needs to be reverse mapped back to the abbreviation
       this.state.current.patient.monitored_address_state = stateOptions.find(state => state.name === this.props.currentState.monitored_address_state)?.abbrv;
     }
-    this.handleChange = this.handleChange.bind(this);
-    this.whereMonitoredSameAsHome = this.whereMonitoredSameAsHome.bind(this);
-    this.validate = this.validate.bind(this);
   }
 
-  handleChange(event) {
+  handleChange = event => {
```

Comment:
I would be okay making this change globally, but it would have to be global. Otherwise stay consistent (or give a really good reason for just doing this to these few files).

Reply:
I was working in this directory (`.../entrollment/steps/xxx.js`) and saw them and figured I could remove a few "bind-the-this" lines, so I decided to change them in the files I was working in. Should I revert these changes in these files?

Reply:
Yeah to Adam's point, this is a point of inconsistency I'm afraid I contributed to. I use the newer way of binding functions (as you did here), and used them throughout the admin refactor so I'm afraid it's already a bit inconsistent throughout the application. I'd personally be okay with keeping the change here (so that at least as we go we can refactor them) and making a ticket to globally go through and make this change as well. At some point we need to decide on frontend styling/conventions as variable name conventions are also inconsistent throughout, but that's out of scope of this obviously.

Reply:
Okay, I'd be good creating a new ticket to do this everywhere, and make sure we start enforcing this from now on (and leave this PR as-is).

Reply:
Ticket made: https://tracker.codev.mitre.org/browse/SARAALERT-829